vertical-tabbrowser.xml

```diff
@@ -786,6 +786,9 @@
         const canvas = document.getAnonymousElementByAttribute(this, 'anonid', 'tab-meta-image');
         canvas.width *= window.devicePixelRatio;
         canvas.height *= window.devicePixelRatio;
+        if (window.VerticalTabs.getFirefoxVersion() >= 55){
+          document.getAnonymousNodes(this)[0].getElementsByClassName('tab-throbber')[0].removeAttribute('layer');
```

Comment:
I would normally complain a lot about this kinda digging into XUL, but in this case we control the XUL, so it should be fine.  🙂